iphone/Classes/TiUIiPadPopoverProxy.m

```diff
@@ -466,13 +427,9 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
     UIEdgeInsets oldInsets = [[change objectForKey:@"old"] UIEdgeInsetsValue];
     NSValue *insetsValue = [NSValue valueWithUIEdgeInsets:newInsets];
 
-    if (!UIEdgeInsetsEqualToEdgeInsets(oldInsets, newInsets)) {
+    if (!UIEdgeInsetsEqualToEdgeInsets(oldInsets, newInsets) || deviceRotated) {
       deviceRotated = NO;
       [self updateContentViewWithSafeAreaInsets:insetsValue];
-    } else if (deviceRotated) {
-      // [self viewController]  need a bit of time to set its frame while rotating
-      deviceRotated = NO;
-      [self performSelector:@selector(updateContentViewWithSafeAreaInsets:) withObject:insetsValue afterDelay:.05];
```

Comment:
Does the safe area still update correctly after rotation?

Reply:
Yes, I deleted the delay and makes updateContentViewWithSafeAreaInsets to waiting for main thread
So I ensured safe area update correctly, This will fix layout issue when you open popover from barButtonItem